.dev/appsettings.override.json

```diff
@@ -86,6 +86,25 @@
             },
             "Application": {
                 "didDomainName": "localhost"
+            },
+            "PublicRelationshipTemplateReferences": {
+                "test": [
+                    {
+                        "title": "a",
+                        "description": "Description",
+                        "truncatedReference": "Reference"
+                    },
+                    {
+                        "title": "b",
+                        "description": "Description",
+                        "truncatedReference": "Reference"
+                    },
+                    {
+                        "title": "c",
+                        "description": "Description",
+                        "truncatedReference": "Reference"
+                    }
+                ]
```

Comment:
hmm, shouldn't we mock this call instead? I'd be a bit careful about defining something here and expecting it to work, b/c when you bump up the backbone soon this will fail. Also now you loose the ability to test the 404 behaviour.